src/www/ui_components/server-list.html

```diff
@@ -66,6 +67,10 @@
         localize: Function,
         rootPath: String,
         servers: Array,
+        hasMultipleServers: {
```

Comment:
Nit: it's a little more readable to make this hasOneServer to avoid the extra negative.

Reply:
Done.